src/main/java/net/uaznia/lukanus/hudson/plugins/gitparameter/GitParameterDefinition.java

```diff
@@ -290,10 +300,12 @@ public String prettyRevisionInfo(GitClient newgit, Revision r) {
 					return Collections.singletonMap(errMsg, errMsg);
 				}
 
-				FetchCommand fetch = newgit.fetch_().from(remoteURL,
-						repository.getFetchRefSpecs());
-				fetch.execute();
 				if (type.equalsIgnoreCase(PARAMETER_TYPE_REVISION)) {
+					// If we use the GitClient#getRemoteReferences function, we don't
```

Comment:
Is there a reason not to do this?

Being limited to the workspace also means that the available branches/tags can be outdated by the time you want to run a build, which means you have to build once just to fetch updated refs, and then build again to use the new tag.